src/httpServer.ts

```diff
@@ -1,15 +1,15 @@
 import fs from "fs";
 import http from "http";
 import https from "https";
-import logger from "./lib/Logger";
 import app from "./app";
+import { logger } from "./lib/logger";
```

Comment:
I think you just need to remove the parenthesis from around 'logger'
When I do that `npm run build` is ok locally

Reply:
done